test/browser_tests/page_objects/page_careers-current-openings.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var _assign = require( 'lodash' ).assign;
+var _getQAelement = require( '../util/QAelement' ).get;
+var careersSocialSection = require( '../shared_objects/careersSocialSection' );
+var relatedLinksSection = require( '../shared_objects/relatedLinksSection' );
+
+
+function CurrentOpenings() {
+
+  _assign( this, careersSocialSection, relatedLinksSection );
+
+  this.get = function() {
+    browser.get( '/careers/current-openings/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+  this.sideNav = element( by.css( '.nav-secondary' ) );
+
+  this.introSection = _getQAelement( 'intro-section' );
+
+  this.introSectionTitle = this.introSection.element( by.css( 'h1' ) );
+
+  this.introSectionLink = this.introSection.element( by.css( 'a' ) );
+
+  this.jobOpeningsTable =
+  element( by.css( '[data-qa-hook="job-openings-section"] table' ) );
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/25449812700f8ec8d3b478bc53f2cad34387e630